products/jbrowse-desktop/src/rootModel.ts

```diff
@@ -531,8 +537,53 @@ export default function rootModelFactory(pluginManager: PluginManager) {
         subMenu.splice(position, 0, menuItem)
         return subMenu.length
       },
+      stopTrackingUndo() {
+        self.history.resumeTrackingUndo()
+      },
+      resumeTrackingUndo() {
+        self.history.resumeTrackingUndo()
+      },
 
       afterCreate() {
+        document.addEventListener('keydown', event => {
+          if (event.shiftKey) {
+            if (
+              (event.ctrlKey && event.key === 'z') ||
+              (event.metaKey && event.key === 'z')
+            ) {
+              if (self.history.canRedo) {
```

Comment:
I don't think this works on my computer (ubuntu,chrome,etc). could we use ctrl+y to redo? seems like a relatively common shortcut for redo

Reply:
I think we should have both shift+meta+z and ctrl+y -- shift+meta+z is redo across the board for macs -- i forgot ctrl+Y was the original redo stroke, it would be good to have it in!

Reply:
should be fine if both